### ISSUE TITLE
Media resources with various encodings

### ISSUE THREAD

**Issue by elf-pavlik** [solid/data-interoperability-panel#33]
If we take an example of photograph or a video recording. I think for each we would have and RDFSource describing it as creative work. It may include information about who took photo / video, geolocaiton, what it depicts, who appears in the creative work (aka 'tagged in' social feature). Also if someone would like or comment, the would reference that 'crative work' which I would consider RDFSource.

At the same time, each one of those creative works would have multiple encodings. For photos, it could have `image/jpeg` version or `image/png`. Also having [Responsive images](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) feature, we can expect even more encodings of the same image. Similar for videos we can expect multiple encodings of the same creative work with different codecs and qualities and frame rates (720p60 etc.)

I'd like to clearly define difference between the conceptual creative work and multiple encodings of it each denoted by distinct URI. I think we also should notice that in some cases we want to have meta data for each of the encodings with information like codec used, encoding software which produced it etc.

**Comment by TallTed** [comment]
Sure.  Something like `http://example.com/movie#this` could be the URI for the work, with `http://example.com/movie.mov`, `http://example.com/movie.mp4`, `http://example.com/movie.avi`, `http://example.com/movie.720p60.mp4`, `http://example.com/movie.1080i60.mp4`, etc., for different encodings, resolutions, etc.

Each of these should have its own description, including details for how derivative encodings are related to the original (and which that is), etc.

The basic description of the creative work (photographer/camera-person, director, scriptwriter, composer, etc.) should be inherited by all the encoding/derivatives, which could happen by various means.

I think most of this is reasonably well covered by previous work (though refinements and improvements are almost certainly possible), significantly within [BIBO](https://www.dublincore.org/specifications/bibo/) and [PROV](https://www.w3.org/TR/prov-primer/).

**Comment by elf-pavlik** [comment]
> Each of these should have its own description, including details for how derivative encodings are related to the original (and which that is), etc.

I understand that each of those encoding resources (Non-RDF Source) would have it's own encoding specific metadata resource (`rel=describedby`?). Then each of that description of *MediaObject* would reference the same *CreativeWork* with something like [`schema:endoding`](https://schema.org/encoding)

> A media object that encodes this CreativeWork. This property is a synonym for associatedMedia.

**Comment by elf-pavlik** [comment]
In https://www.w3.org/TR/cooluris/#linking I notice
> The HTML document at http://www.example.com/people/alice should contain in its header a `<link>` element that points to the corresponding RDF document:

```html
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
  <head>
    <title>Alice's Homepage</title>
    <link rel="alternate" type="application/rdf+xml"
          title="RDF Representation"
          href="http://www.example.com/data/alice" />
  </head> ...
```

> This allows RDF-aware Web clients to discover the RDF information. The approach is [recommended in the RDF/XML specification ](http://www.w3.org/TR/2004/REC-rdf-syntax-grammar-20040210/#section-rdf-in-HTML) ([RDFXML], section 9). If the RDF data is about the Web page, rather than an expression of the information in it, then we recommend using `rel="meta"` instead of `rel="alternate"`.

Above doesn't even mention `rel="describedby"`, I think it would help if we provide guidance on when to use:
* describedby
* meta
* alternate

**Comment by ericprud** [comment]
Here's what I dug up:

**describedby**: [RFC5988](https://tools.ietf.org/html/rfc5988#section-6.2.2) turfs the definition of `describedBy` to [POWDER](https://www.w3.org/TR/powder-dr/#appD), which says:
> The relationship A 'describedby' B asserts that resource B provides a description of resource A. There are no constraints on the format or representation of either A or B, neither are there any further constraints on either resource.

**meta**: microformats [loosely defines](http://microformats.org/wiki/rel-meta) meta as:
> The meta link relationship is used to link to a resource containing external metadata about the HTML page. The metadata is often assumed to be in RDF format, as indicated by the type attribute.

As far as I can see, there's no relationship to the [HTML `meta` element](https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element). Also, the heritage of `meta` may go back to a [W3C QA doc](https://www.w3.org/QA/Tips/use-links) with text from before 2000.

**alternate**: This does come from the [HTML spec](https://html.spec.whatwg.org/multipage/links.html#rel-alternate) and has some special cases:
> rel attribute also contains the keyword stylesheet ...
> type attribute set to the value `application/rss+xml` ...

and the fall-through applies to our case:
> Otherwise The keyword creates a hyperlink referencing an alternate representation of the current document

The [IANA Link Relations Registry](https://www.iana.org/assignments/link-relations/link-relations.xhtml) has no entry for `meta`. (Beware, the results of an [IETF search for `link`](https://tools.ietf.org/id/link) are almost all related to routing and OSPF.)

I'm guessing we want `describedby` unless we want to write a (badly-needed) RFC for `meta` ([analogous RFC](https://tools.ietf.org/id/draft-hamnaberg-publish-link-relation-01.txt), [source](https://tools.ietf.org/id/draft-hamnaberg-publish-link-relation-01.xml)).

**Comment by elf-pavlik** [comment]
> I'm guessing we want `describedby`

Makes sense for 'resource description', unless we find advantage in using URIs for all the relations and have a clear way to express them in RDF (eg. when exporting all the metadata).

I think `alternate` could be used in cases which require independent operations on different representations, like in https://github.com/solid/specification/issues/69#issuecomment-573129043

**Comment by TallTed** [comment]
See also https://github.com/solid/data-interoperability-panel/issues/37#issuecomment-573761797
